app/models/user.rb

```diff
@@ -1,2 +1,13 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: :author_id
+  has_many :comments, foreign_key: :author_id
+  has_many :likes
```

Comment:
- [Optional] Hi @codeWithHyder, I notice the likes table has two foreign keys referencing the user's table: user_id and author_id.